src/sim_recon/images/tiff.py

```diff
@@ -101,25 +117,26 @@ def get_channel_dict(channel: ImageChannel) -> dict[str, Any] | None:
         "metadata": {},
     }
 
-    if xy_pixel_size_microns is not None and None not in xy_pixel_size_microns:
-        xy_pixel_size_microns = cast(tuple[float, float], xy_pixel_size_microns)
+    if resolution is not None:
         # TIFF tags:
         tiff_kwargs["resolution"] = (
-            1e4 / xy_pixel_size_microns[0],
-            1e4 / xy_pixel_size_microns[1],
+            1e4 / resolution.x,
+            1e4 / resolution.y,
         )
         tiff_kwargs["resolutionunit"] = (
             tf.RESUNIT.CENTIMETER
         )  # Use CENTIMETER for maximum compatibility
 
     if ome:
-        if xy_pixel_size_microns is not None:
+        tiff_kwargs["metadata"]["Name"] = output_path.name
```

Comment:
Stylistically, I would define `tiff_metadata = tiff_kwargs["metadata"]` as Python is interpreted

Reply:
Okay, I've changed it to avoid the double indexing